frontend/src/hooks/useArchive.ts

```diff
@@ -0,0 +1,14 @@
+import {useEffect, useState} from "react";
+import {NasaPicture} from "../model/NasaPicture";
+import {getArchive} from "../service/api-service";
+
+export default function useArchive(){
+    const [archivePictures, setArchivePictures] = useState<NasaPicture[]>();
+
+    useEffect(() =>{
+        getArchive()
+            .then(archivePictures => setArchivePictures(archivePictures))
+            .catch(()=> console.error("Can't show Archive"))
+    }, [])
+    return archivePictures
+}
```

Comment:
Bei JS Datei wird dieses Symbol (wie in der Zeile 14) angezeigt, wenn man keine neue Zeile am Ende einer .ts Datei macht. Wenn Du das weg haben willst, kannst einfach Enter am Ende jeder .ts Datei machen. Das verschwindet dann!